src/util/format_type.cpp

```diff
@@ -89,6 +89,14 @@ std::ostream &format_rec(std::ostream &os, const typet &type)
     return os << "floatbv[" << to_floatbv_type(type).get_width() << ']';
   else if(id == ID_c_bool)
     return os << "c_bool[" << to_c_bool_type(type).get_width() << ']';
+  else if(id == ID_bool)
+    return os << "\xf0\x9d\x94\xb9"; // u+1D539, 'B'
```

Comment:
Is that UTF-8 encoding? If so you might like to use `u8"\U0001D539"` in future rather than directly quote the encoded form

Reply:
(`\uNNNN` for BMP characters like the three below)